scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
> Under the current application design, there is no way to roll back these types of changes.

Well we still take regular backups of the database, on a weekly basis I think, don't we? As we no longer have the delete functionality through the GUI there's no other way around it, and back then this was also a risky option as people could also delete things inadvertently, see #202. Clearly the best solution to avoid these kind of risks would be to implement the soft delete solution proposed in https://github.com/aodn/aatams/issues/204